packages/server/file-store-api/main.go

```diff
@@ -77,13 +78,13 @@ func main() {
 
 			multipartFileHeader, err := c.FormFile("file")
 			if err != nil {
-				c.AbortWithStatus(500) //todo
```

Comment:
c ?

Reply:
c is the context, it is pretty much standard in our entire codebase